cluster-autoscaler/cloudprovider/aws/examples/cluster-autoscaler-autodiscover.yaml

```diff
@@ -157,7 +157,7 @@ spec:
             - --node-group-auto-discovery=asg:tag=k8s.io/cluster-autoscaler/enabled,k8s.io/cluster-autoscaler/<YOUR CLUSTER NAME>
           volumeMounts:
             - name: ssl-certs
-              mountPath: /etc/ssl/certs/ca-certificates.crt
+              mountPath: /etc/ssl/certs/ca-bundle.crt
```

Comment:
I need to determine whether the previous default `ca-certificates.crt` was used for a specific reason. If that is, for example, the correct path on kops-installed clusters on EC2, then I think it would be better to add a new `cluster-autoscaler-autodiscover-eks.yaml` file instead of modifying this one.

Reply:
@jaypipes As you defined the deployment volume host path is "/etc/ssl/certs/ca-bundle.crt" as below:
```
      volumes:
        - name: ssl-certs
          hostPath:
            path: "/etc/ssl/certs/ca-bundle.crt"
```

So, to solve this issue, I would think that we are required to update the volume mount path to "/etc/ssl/certs/ca-bundle.crt" in

```
 volumeMounts:
            - name: ssl-certs
              from "mountPath: /etc/ssl/certs/ca-certificates.crt"
              to     "mountPath: /etc/ssl/certs/ca-bundle.crt"
```

 to match the volume you defined. In my opinion, it is not about EKS or KOPS right now.

Reply:
From what I can see, it looks like the location of the SSL certs primarily has to do with the operating system. [Here's the kops guidance for setting up cluster autoscaler.](https://github.com/kubernetes/kops/blob/9885df83ad1dc0e07e921d9864f25c696cba5a77/addons/cluster-autoscaler/README.md#L17)

Perhaps we should just follow their example and just add a comment? As you can see [here](https://serverfault.com/questions/62496/ssl-certificate-location-on-unix-linux), looks like there's a lot of variation across Linux distros. Even in the EKS case, if you set up non-AmazonLinux worker nodes, the cert could be at a different location.

Reply:
@mmerkes So, could we add another comment in the aws example, so anyone would know if they use EKS with Amazon Linux, they should config like this.

Reply:
@dmakeroam Makes sense, though I believe it would be any AmazonLinux worker nodes, not just EKS.

Reply:
@mmerkes Agreed with this